tests/json/test_json.py

```diff
@@ -48,7 +47,6 @@ def test_json_results_equivalence(
 
     # reset random seed
     seed = 0
-    random.seed(seed)
     np.random.seed(seed)
     pnl.core.globals.utilities.set_global_seed(seed)
```

Comment:
This line should stay, the other resets are OK to remove

Reply:
Sorry, thought the comment pointed to the line. Line 53/51 (`pnl.core.globals.utilities.set_global_seed(seed)`) should stay.